sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -279,40 +279,36 @@ class SparkSessionExtensionSuite extends SparkFunSuite with SQLHelper {
     }
     withSession(extensions) { session =>
       session.conf.set(SQLConf.ADAPTIVE_EXECUTION_ENABLED, enableAQE)
-      Seq(true, false).foreach { enableEvaluator =>
```

Comment:
According to the previous discussion, we don't need to test it as we are already using the evaluator by default, and we can test `RDD#mapPartitionsWithEvaluator` when we use it by default.